deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java

```diff
@@ -41,25 +34,33 @@ public abstract class OpenApiGeneratorCodeGenBase implements CodeGenProvider {
 
     private static final String DEFAULT_PACKAGE = "org.openapi.quarkus";
 
+    // It will be ignored if INPUT_BASE_DIR is specified
     @Override
     public String inputDirectory() {
         return "openapi";
     }
 
+    @Override
+    public boolean shouldRun(Path sourceDir, Config config) {
```

Comment:
I'm not sure it's a good idea to silently return `false` here.
Maybe we could throw an exception to break compilation.
WDYT @ricardozanini?

Reply:
Yes, let's throw an exception instead.

Reply:
Yes.